packages/equation_extraction/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN ./gradlew bootJar
 WORKDIR /
 
 # Copy the jar to the root directory
-RUN mv /taskrunner/build/libs/*.jar .
+RUN cp /taskrunner/build/libs/*.jar .
```

Comment:
this is the fix, copy the jar files, dont move them

Reply:
how come we need to copy in this build but not in the others?